packages/editor/src/components/document-tools/index.js

```diff
@@ -139,7 +140,11 @@ function DocumentTools( { className, disableBlockTools = false } ) {
 					<>
 						{ isLargeViewport && ! hasFixedToolbar && (
 							<ToolbarItem
-								as={ ToolSelector }
+								as={
+									window.__experimentalSimpleEditingMode
+										? SimpleEditingModeSelector
+										: ToolSelector
```

Comment:
I'm not sure why we're using a separate too selector personally, can't we reuse the same?

Reply:
I think this was just for the purposes of getting an experimnet up without editing the other component. We should be able to reuse the `ToolSelector` - especially if we use `Select` mode.

Reply:
`editorMode` (navigationMode`) makes sense to me, since I imagine eventually this will be used for a wider range of post types.

One slightly awkward thing is the current modes are `edit` and `navigation` (where edit is the normal mode), and it sounds like the new public names for the modes will be `design` and `edit` (where edit is the `contentOnly` style mode). 😬 

We could either keep the existing names and live with it forever being confusing or deprecate them.

Reply:
It had to be a new component so that it could live in `packages/editor` where rendering mode is (not `packages/block-editor` where editing mode is) and so that we could switch it when the experiment is enabled/disabled. If we're okay with using editor mode (see other thread) then yeah we can re-use the existing component.